File: Cmdlets/Models/BankExchangeRate.vb
Imports TIKSN.Finance

Public Class BankExchangeRate

	Public Sub New(BankName As String, CurrencyPair As CurrencyPair, Rate As Decimal)
		Me.BankName = BankName
		Me.CurrencyPair = CurrencyPair
		Me.Rate = Rate
	End Sub

	Public ReadOnly Property BankName As String

	Public ReadOnly Property CurrencyPair As CurrencyPair

	Public ReadOnly Property Rate As Decimal
End Class

File: C#/Cmdlets/Cmdlets/Models/BankExchangeRate.cs
using TIKSN.Finance;

namespace TIKSN.Cmdlets.Models
{
    public class BankExchangeRate
    {
        public BankExchangeRate(string BankName, CurrencyPair CurrencyPair, decimal Rate)
        {
            this.BankName = BankName;
            this.CurrencyPair = CurrencyPair;
            this.Rate = Rate;
        }

        public string BankName { get; }

        public CurrencyPair CurrencyPair { get; }

        public decimal Rate { get; }
    }
}